### ISSUE TITLE
"Logging in..." Indefinitely

### ISSUE THREAD

**Issue by computerweekend** [kruppel/soundcloud#21]
Version: Songbird 1.9.3, Build 1959 (20110127030554)
Soundcloud app version 1.0.1
Win7 64-bit

Start Songbird.
Click the login to Soundcloud button on the very bottom right hand corner of Songbird.
Log in with correct credentials (verified in browser, on the Soundcloud website).
Login window reports that it is "Logging in..." and the icon spins.
Soundcloud connect window pops up on left side of screen, and asks if you want to connect Soundcloud to Songbird.
Click connect. Connection is reported as successful - this can be verified by logging in to Soundcloud via browser.
Meanwhile, the small login window is still "Logging in..." and the icon is spinning.
Closing the Soundcloud connect window cancels the connection.
Never logs in to Soundcloud.

I have tried several uninstalls and reinstalls of both Songbird and the Soundcloud app, disabling then uninstalling all other apps and a profile wipe. Nothing seems to fix it. Any ideas? Thanks in advance.

**Comment by kruppel** [comment]
Thanks for the report. Are there any errors dumped to the error console?

**Comment by computerweekend** [comment]
Sorry for not including that before, but yes, a lot. So many, in fact, that the error console has to cut some off! Exciting!

Warning: Unknown property 'border-radius'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1400

Warning: Unknown property 'border-bottom-right-radius'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1402

Warning: Unknown property 'border-top-left-radius'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1402

Warning: Error in parsing value for 'background'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1402

Warning: Error in parsing value for 'filter'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1402

Warning: Unknown property 'zoom'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1402

Warning: Unknown property 'text-overflow'.  Declaration dropped.
Source File: https://soundcloud.com/stylesheets/general.css?af2741b
Line: 1836

(There are easily hundreds more just like those)

Warning: Unknown pseudo-class or pseudo-element 'submit'.
Source File: https://soundcloud.com/oauth/authorize
Line: 0

(There's one of these for 'submit', 'reset', 'button', 'radio', 'checkbox', 'has', 'text', and a ton for 'first')

Warning: The 'charCode' property of a keydown event should not be used. The value is meaningless.
Source File: https://soundcloud.com/oauth/authorize
Line: 0

And that's the last one at the bottom of the console.

If you need the entire log, I would be happy to provide everything I am able to.

**Comment by kruppel** [comment]
These are warnings. There should be an option to only display errors. If you could paste them, that'd be helpful.

Once you've logged in successfully, the authorization dialog window should close. Is that not happening? If so, then could I see a screenshot - the dialog should be displayed, so don't escape it here - of what the window looks like once you're logged in? 

Also, could you by chance capture and post your profile folder? See http://wiki.songbirdnest.com/Docs/Songbird_Support_Aids/Capturing_Your_Songbird_Profile for more information. There's no option to create an attachment here, so just email it to me at kurt-at-songbirdnest.com

**Comment by computerweekend** [comment]
Sorry for the confusion - there are no actual errors, just those warnings. I'm a little new to Songbird and apparently can't read =P

And yes, the authorization window does indeed not close. I will send both a screenshot of what I see and my profile folder as well. Thank you so much for your help.

Actually, one additional thing: The zip containing my profile is huge... as in... gmail won't let me mail it (50mb). Is there something in specific you need? Like the soundcloud@songbirdnest.com folder? Or is it more complex than that?

**Comment by kruppel** [comment]
The entire profile would be good to have, so as to be able to debug from a baseline. If it's too much of a hassle, I may be able to do without it depending on what the screenshot looks like. So, long story short - let's start with a screenshot of the auth window and then we can see where to go from there.

Thanks again.

**Comment by kruppel** [comment]
Bah. SoundCloud made some changes to the auth pages. Fix is on the way.

**Comment by computerweekend** [comment]
Oh, awesome :] Thank you so much for looking into this for me, I really love this addon! Still need me to upload files to you? As for the profile, I could always break it up into a few rar files and send it that way if you still need to see it!

**Comment by kruppel** [comment]
Uploaded a new version to http://addons.songbirdnest.com/addons/1964. If you still run into trouble, feel free to let me know. Cheers.

**Comment by computerweekend** [comment]
Edit: Well, nevermind! Maybe authentication was down for 2 hours? It just re-authenticated out of nowhere and seems to be fine now. I'm going to go ahead and not question it for now. Sorry! =)

Hello again! Things were working awesomely, I was able to log in and stayed logged in for a couple of hours, with the ability to listen to music and view my favorites and such... and now all of a sudden I can't log in, and no tracks are showing up at all - the radio viewer is showing "no tracks found". Before the authentication issue was fixed, I was at least able to listen to soundcloud. I've checked the Soundcloud website and I am able to log in, view and listen to tracks just fine, so I think I can safely assume that they're not down or in maintenance.

I have tried uninstalling and reinstalling the mod, and restarting Soundcloud. I'm able to log into last.fm, and have Songbird allowed through my firewall. I'm not running any software that would otherwise block a connection to Soundcloud or Songbird.

At least the authentication window doesn't come up anymore, I just get the spinning icon.
